tests/e2e/helpers/api/api-database.ts

```diff
@@ -1,253 +1,341 @@
+                certificate: databaseParameters.clientCert!.certificate,
+                key: databaseParameters.clientCert!.key,
+            };
+        }
+        const response = await sendRequest(
+            Methods.post,
+            '/databases',
+            201,
+            requestBody
+        );
+        await t
+            .expect(await response.body.name)
+            .eql(
+                databaseParameters.databaseName,
+                `Database Name is not equal to ${databaseParameters.databaseName} in response`
+            );
     }
-}
 
-/**
- * Add a new database from OSS Cluster through api using host and port
- * @param databaseParameters The database parameters
- */
-export async function addNewOSSClusterDatabaseApi(databaseParameters: OSSClusterParameters): Promise<void> {
-    const requestBody = {
-        'name': databaseParameters.ossClusterDatabaseName,
-        'host': databaseParameters.ossClusterHost,
-        'port': Number(databaseParameters.ossClusterPort)
-    };
-    const response = await sendRequest(Methods.post, '/databases', 201, requestBody);
-    await t.expect(await response.body.name).eql(databaseParameters.ossClusterDatabaseName, `Database Name is not equal to ${databaseParameters.ossClusterDatabaseName} in response`);
-}
+    /**
+     * Add a new Standalone databases through api using host and port
+     * @param databasesParameters The databases parameters array
+     */
+    async addNewStandaloneDatabasesApi(
+        databasesParameters: AddNewDatabaseParameters[]
+    ): Promise<void> {
+        if (databasesParameters.length) {
+            databasesParameters.forEach(async (parameter) => {
+                await this.addNewStandaloneDatabaseApi(parameter);
+            });
+        }
+    }
 
-/**
- * Add a Sentinel database via autodiscover through api
- * @param databaseParameters The database parameters
- * @param primaryGroupsNumber Number of added primary groups
- */
-export async function discoverSentinelDatabaseApi(databaseParameters: SentinelParameters, primaryGroupsNumber?: number): Promise<void> {
-    let masters = databaseParameters.masters;
-    if (primaryGroupsNumber) {
-        masters = databaseParameters.masters!.slice(0, primaryGroupsNumber);
+    /**
+     * Add a new database from OSS Cluster through api using host and port
+     * @param databaseParameters The database parameters
+     */
+    async addNewOSSClusterDatabaseApi(
+        databaseParameters: OSSClusterParameters
+    ): Promise<void> {
+        const requestBody = {
+            name: databaseParameters.ossClusterDatabaseName,
+            host: databaseParameters.ossClusterHost,
+            port: Number(databaseParameters.ossClusterPort),
+        };
+        const response = await sendRequest(
+            Methods.post,
+            '/databases',
+            201,
+            requestBody
+        );
+        await t
+            .expect(await response.body.name)
```

Comment:
can we also split SendRequest method to Put, Get and delete as separate method?
and I am bot sure about verifying status code in method. on one hand it is good to understand immediately that bd is not correct, on the other hand it is violate single responsibility
probably we can move the verification on the 1 layer above
what do you think?